### DISCUSSION TITLE
Best practice to avoid corrupt data after deploy?

### DISCUSSION THREAD

**Discussion by andonovn** [livewire/livewire#4311] [1. Help]
Hey there,

Goal is to remove a "reactive" (public) property which is bound (wire:model) to an input element. Once deployed that caused checksums to be invalid, hence corrupt data exceptions started popping up.

I'll admin I haven't spent much time figuring that out, I wonder if I am missing something obvious? Are there any best practices that I am missing?

Two solutions in my head, none of them feels ideal, tho (also, none of them tested if they actually work):
 - Keeping the public property on for bc reasons (just remove the input element and deploy)
 - Forcing a page refresh for active users (similar to what most SPAs do)

I've already read a lot on the topic but couldn't find any advice. I believe I listened Caleb expressing his thoughts on that subject on some podcast episode but I couldn't find the episode either. Any chance anyone remember?

I'll also admit that it may be a complete coincidence that I experience the corrupt data error with the deploy, it may be something completely unrelated. So feel free to point out that if you feel that removing that public property shouldn't cause that issue at first place.

**Comment by joshhanley** [comment]
@andonovn yeah at the moment there isn't a great solution, but we're working on that. There was a discussion #3954 about this, so maybe if you could add your comments to that, so it's all in the same place, that would be great!

Thanks!
